sdcm/cluster.py

```diff
@@ -1606,6 +1606,34 @@ def datacenter_setup(self, datacenters):
         cmd = cmd.format(datacenters[self.dc_idx], dc_suffix)
         self.remoter.run(cmd)
 
+    def patch_scylla_yaml(self, yaml_file=SCYLLA_YAML_PATH, **kwargs):
+        """
+        A user can send a dictionary of values ​​(just like the function "config_setup") and thus only change the
+        values he has chosen. If one of the keys contains a "None" value, the same value will be removed from the Yaml
+        file.
+        """
+        yaml_dst_path = os.path.join(tempfile.mkdtemp(prefix='scylla-longevity'), 'scylla.yaml')
```

Comment:
you are missing a copy here, i.e. step 1. :
1) copy from node to test
2) load , make changes, save
3) copy from test to node

Reply:
Fixed